docs/docs/concepts/01-intro.md

```diff
@@ -13,9 +13,9 @@ between different parts of the AICA software stack.
 
 <SystemOverview className="themedSVG" style={{width: "100%"}}/>
 
-## AICA framework
+## AICA Core
 
-The AICA **framework** is a virtual robotics workspace pre-configured with a growing collection of software libraries
+**AICA Core** is a virtual robotics workspace pre-configured with a growing collection of software libraries
 for motion generation, signal processing, machine learning and control algorithms. It includes hardware interfaces for
 real-time external control of popular robot brands, force-torque sensors and cameras.
 The workspace is the foundation and the software libraries are the building blocks.
```

Comment:
You can replace line 23 with `<!-- <ComponentsControllersHardware className="themedSVG" style={{width: "100%"}}/> -->`, this comments out the include of the picture